docbook5-book/xml/common_intro_feedback.xml

```diff
@@ -15,8 +15,8 @@
  <title>Giving feedback</title>
```

Comment:
```suggestion
 <title>Improving the documentation</title>
```

Reply:
Could we also change that to what we have in doc-sle right now? Sounds more active to me than 'Giving feedback'.